proto/src/determined/trial/v1/trial.proto

```diff
@@ -84,6 +84,8 @@ message Trial {
   // The wall clock time is all active time of the cluster for the trial,
   // inclusive of everything (restarts, initiailization, etc), in seconds.
   double wall_clock_time = 12;
+  // The sum of sizes of all resources in all checkpoints for the trial.
+  uint64 total_checkpoint_size = 13;
```

Comment:
**question:** I see this defined here as `uint64` but the endpoint returns this as a string for some reason:
![Screen Shot 2022-03-01 at 9 11 55 PM](https://user-images.githubusercontent.com/220971/156293779-6b7906d6-311e-4df4-ae20-579bc9716be0.png)